crates/synthesis/src/two_qubit_decompose.rs

```diff
@@ -2119,165 +2203,87 @@ impl TwoQubitBasisDecomposer {
             .collect()
     }
 
-    /// Decompose a two-qubit ``unitary`` over fixed basis and :math:`SU(2)` using the best
-    /// approximation given that each basis application has a finite ``basis_fidelity``.
+    /// Synthesizes a two qubit unitary matrix into a :class:`.DAGCircuit` object
+    ///
+    /// Args:
+    ///     unitary (ndarray): A 4x4 unitary matrix in the form of a numply complex array
+    ///         representing the gate to synthesize
+    ///     basis_fidelity (float): The target fidelity of the synthesis This is a floating point
+    ///         value between
```

Comment:
Seems like something happened here with your editor.

Reply:
Or my brain turned off while I was writing the sentence. I'll finish the thought and push an update.

Reply:
Fixed in: https://github.com/Qiskit/qiskit/pull/14650/commits/0c0fcc0dc9ad6fb35b71db73fb538c8d92a43b55